content/resources/operator/_index.md

```diff
@@ -378,3 +378,15 @@ resources:
 	- my_program-TEST1_name3_downloader
 	- my_program-TEST1_name4_downloader
 ```
+
+## 8. ETL and Data Explorer Configurations
+
+ETL stands for “extract, transform, load”. In Gen3, it refers to the process of moving data from the PostgreSQL database (graph model) to the ElasticSearch database (flat model), which is carried out by the Gen3 Framework Service [Tube](https://github.com/uc-cdis/tube).
+
+Newly ingested data to the Sheepdog Service can be queried immediately via [Peregrine](https://github.com/uc-cdis/peregrine), but not on the Data Explorer, which is powered by [Guppy](https://github.com/uc-cdis/guppy) on the backend.  During the ETL process, Tube will populate ElasticSearch indices and Guppy makes the ElasticSearch indices available for queries for the Data Explorer.
+
+In practice, Guppy/Tube need to be configured with the ElasticSearch indices in the manifest.json (versions in the [versions block](https://github.com/uc-cdis/cdis-manifest/blob/6bc0dc84bec2af7d7971d2a4342319b225728969/gen3.datacommons.io/manifest.json#L23) and indices in the [guppy block](https://github.com/uc-cdis/guppy#configurations)) and the [etlMapping.yaml file](https://github.com/uc-cdis/tube/blob/master/docs/configuration_file.md) has to be configured to list those indices.
```

Comment:
we should also mention that `aws-es-proxy` is usually needed in this case althought it is not a Gen3 service, unless you can provide a customized endpoint to access ES